titleToNumber.js

```diff
@@ -0,0 +1,38 @@
+const titleToNumber = (columnTitle) => {
+  const alphabet = [
+    "A",
+    "B",
+    "C",
+    "D",
+    "E",
+    "F",
+    "G",
+    "H",
+    "I",
+    "J",
+    "K",
+    "L",
+    "M",
+    "N",
+    "O",
+    "P",
+    "Q",
+    "R",
+    "S",
+    "T",
+    "U",
+    "V",
+    "W",
+    "X",
+    "Y",
+    "Z",
+  ];
+  const { length } = alphabet;
+  let result = 0;
+  let letterIndex = 0;
+  for (let i = columnTitle.length - 1; i >= 0; i -= 1) {
+    result += length ** i * (alphabet.indexOf(columnTitle[letterIndex]) + 1);
```

Comment:
Lets move this code `alphabet.indexOf(columnTitle[letterIndex]) + 1` into appropriately named variable.

Reply:
ready on new commit - ccb3e05b1972b6c352a746bda339c97d2ba050c6